helm-charts/daskhub/values.yaml

```diff
@@ -177,38 +196,118 @@ dask-gateway:
                     chars.append(escaped_hex_char)
             return u''.join(chars)
 
+        # Decide on available instance types and their resource allocation
+        # choices to expose based on cloud provider. For each daskhub hub
+        # managed by 2i2c, there should be these instance types available.
+        #
+        cloud_provider = os.environ["BASEHUB_K8S_DIST"] # gke, eks, or aks
+        instance_types = {
+            "gke": ["n2-highmem-16"],
+            "eks": ["r5.4xlarge"],
+        }
```

Comment:
Yes we don't have any AKS daskhubs yet so I opted to avoid trying to pre-optimize this for now. Its better that it breaks than is setup without this for the first future AKS daskhub anyhow I think.